src/jit/emit.h

```diff
@@ -885,14 +885,16 @@ class emitter
         void checkSizes();
 
         union idAddrUnion {
-            // TODO-Cleanup: We should really add a DEBUG-only tag to this union so we can add asserts
-            // about reading what we think is here, to avoid unexpected corruption issues.
+// TODO-Cleanup: We should really add a DEBUG-only tag to this union so we can add asserts
+// about reading what we think is here, to avoid unexpected corruption issues.
 
+#ifndef _TARGET_ARM64_
             emitLclVarAddr iiaLclVar;
```

Comment:
- I didn't move for 32 bit architectures since it would grow the union.
- I didn't grow for amd64, because the struct I was putting it in was not used to amd64.

Not sure how to proceed